sdk/src/main/java/com/mapbox/maps/MapController.kt

```diff
@@ -325,6 +328,7 @@ internal class MapController : MapPluginProviderDelegate, MapControllable {
       "Add %s plugin dependency to the classpath take automatically load the plugin implementation."
     private const val VIEW_HIERARCHY_MISSING_TEMPLATE =
       "%s plugin requires a View hierarchy to be injected, plugin is ignored."
+    private var assetsManagerNeedInit = true
```

Comment:
should `assetsManagerNeedInit` be static?

Reply:
when such bool variable is checked only once I think it looks better when we check it for `true` and not like 
`if (!assetsManagerInitialized) {...}`

Reply:
`assetsManagerNotInitialized` 😄

ok, up to you